### PR TITLE
More admin commands

### PR DESCRIPTION
**Pull request by xen-000**
- Added !goto, !bring, !unmute, !ungag, !hsay
- !map now has a 5 second delay
- Added a template admin config that gets packaged when building

### CODE REVIEW

File: src/adminsystem.cpp [hunk 1]
```diff
@@ -123,76 +189,251 @@ CON_COMMAND_CHAT(ban, "ban a player")
...
 {
 	if (!player)
 		return;
 
 	int iCommandPlayer = player->GetPlayerSlot();
 
-	ZEPlayer* pPlayer = g_playerManager->GetPlayer(player->GetPlayerSlot());
+	ZEPlayer *pPlayer = g_playerManager->GetPlayer(player->GetPlayerSlot());
 
 	if (!pPlayer->IsAdminFlagSet(ADMFLAG_BAN))
```
Comment (Poggicek) [inline src/adminsystem.cpp:229]: imo removing infractions should require unban flag

File: src/adminsystem.cpp [hunk 2]
```diff
@@ -123,76 +189,251 @@ CON_COMMAND_CHAT(ban, "ban a player")
...
+CON_COMMAND_CHAT(ungag, "ungags a player")
+{
+	if (!player)
+		return;
+
+	int iCommandPlayer = player->GetPlayerSlot();
+
+	ZEPlayer *pPlayer = g_playerManager->GetPlayer(player->GetPlayerSlot());
+
+	if (!pPlayer->IsAdminFlagSet(ADMFLAG_BAN))
```
Comment (Poggicek) [inline src/adminsystem.cpp:379]: ditto

File: src/commands.cpp [hunk 3]
```diff
@@ -287,6 +272,22 @@ CON_COMMAND_CHAT(message, "message someone")
 	UTIL_SayTextFilter(filter, buf, nullptr, 0);
 }
 
+CON_COMMAND_CHAT(say, "say something using console")
+{
+	ClientPrintAll(HUD_PRINTTALK, "%s", args.ArgS());
```
Comment (Poggicek) [inline src/commands.cpp:277]: missing admin check

File: src/commands.cpp [hunk 4]
```diff
@@ -287,6 +272,22 @@ CON_COMMAND_CHAT(message, "message someone")
...
+}
+
+CON_COMMAND_CHAT(takemoney, "take your money")
```
Comment (Poggicek) [inline src/commands.cpp:280]: missing admin check, players could give themselves money

**Review by Poggicek — CHANGES_REQUESTED** [review]

File: src/commands.cpp (hunk 4 above)
Comment (xen-000) [reply src/commands.cpp:280]: Same here

File: src/commands.cpp (hunk 3 above)
Comment (Poggicek) [reply src/commands.cpp:277]: ah right, didn't notice

**Review by Poggicek — APPROVED** [review]

**Pull request merged**